lib/api.go

```diff
@@ -38,16 +38,16 @@ const (
 	// AEIPFS is the IPFS endpoint
 	AEIPFS = APIEndpoint("/ipfs/{path:.*}")
 
-	// profile enpoints
-
-	// AEMe is the "own" profile endpoint
-	AEMe = APIEndpoint("/me")
-	// AEProfile is an alias for the me endpoint
-	AEProfile = APIEndpoint("/profile")
-	// AEProfilePhoto is an endpoint to serve the profile photo
-	AEProfilePhoto = APIEndpoint("/profile/photo")
-	// AEProfilePoster is an endpoint to serve the profile poster
-	AEProfilePoster = APIEndpoint("/profile/poster")
+	// profile endpoints
+
+	// AEGetProfile is an alias for the me endpoint
+	AEGetProfile = APIEndpoint("/profile")
+	// AESetProfile is an endpoint to set the profile
+	AESetProfile = APIEndpoint("/profile/set")
+	// AESetProfilePhoto is an endpoint to set the profile photo
+	AESetProfilePhoto = APIEndpoint("/profile/photo/set")
+	// AESetPosterPhoto is an endpoint to set the profile poster
+	AESetPosterPhoto = APIEndpoint("/profile/poster/set")
```

Comment:
Renamed for clarity/uniformity `AEProfile` -> `AEGetProfile`
`AESaveProfile` -> `AESetProfile`

removed `AEProfilePhoto` and `AEPosterPhoto`, added `AESetProfilePhoto` & `AESetPosterPhoto`

Reply:
Im going to cry on my next rebase :D 
@b5 I share the same confusion here now.

Reply:
I KNOW I'M SORRYYYYYY